techs/zetapack/factions/elves/units/sanctuary/sanctuary.xml

```diff
@@ -76,7 +76,7 @@
 			<ep-cost value="0" />
 			<speed value="300" />
 			<anim-speed value="300" />
-			<animation path="models/sanctuary.g3d" />
+			<animation path="models/sanctuary_con.g3d" />
```

Comment:
This appears to be a duplicate of a previous commit.

Reply:
It shouldn't affect the changing date, the pull request was just coupled with the trojan horse one,

Reply:
@MirceaKitsune once in a while is no problem, but if if duplicate commits aren't held in check, then the commit log will wind up with tons of useless data. Most repo owners have standards to keep that from happening. The pull requests procedure outlined on https://github.com/ZetaGlest/zetaglest-source/blob/develop/CONTRIBUTING.md#pull-requests is designed to prevent duplicate commits and merge conflicts, and is generally the preferred method to do pull requests on most GitHub repos.

Reply:
>and this happens regularly for me, cuz they have to confirm a pull request before I can send another one,

@KeithSammut That's not why it happens. It's because you don't create separate branches. As I've repeated explained. When you submit a patch from a branch you've already altered, it gets combined and mixed up.

Reply:
I see. I guess that for something this small, it shouldn't be an issue this once. Just going to highlight @KeithSammut to be sure he is aware of this for future reference.

Reply:
yeah Mathu gave me some commands to clone another branch (example: master) and upload the changes on it (on the Master Branch)